lib/serverless/aws-lambda.js

```diff
@@ -114,6 +114,9 @@ class AwsLambda {
       return handler
     }
 
+    const isStreamHandler = handler[Symbol.for('aws.lambda.runtime.handler.streaming')] === 'response'
+    // TODO: add supportability metric for streaming?
```

Comment:
are you going to create a metric for this?

Reply:
I'm adding one in the next commit, unless we don't want to record this at all.